src/plugins/visualizations/server/saved_objects/visualization.ts

```diff
@@ -20,9 +20,6 @@ export const visualizationSavedObjectType: SavedObjectsType = {
     getTitle(obj) {
       return obj.attributes.title;
     },
-    getEditUrl(obj) {
```

Comment:
Allows us to decouple the loaders from the saved objects management page and still render the underlying saved object in the SOM Inspect view.